src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -32,13 +38,21 @@
 import io.flutter.console.FlutterConsoleHelper;
 import io.flutter.dart.DartPlugin;
 import io.flutter.utils.FlutterModuleUtils;
+import org.jdom.Element;
+import org.jdom.JDOMException;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import java.io.File;
+import java.io.IOException;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 public class FlutterSdk {
   public static final String FLUTTER_SDK_GLOBAL_LIB_NAME = "Flutter SDK";
+
+  private static final String SDK_LIBRARY_PATH = ".idea/libraries/Dart_SDK.xml";
```

Comment:
Thanks! I will ask if I can't figure it out.

I'll note that the results of reading source code and experiments may go out of date too (in the next release, if the source code changes). Javadoc doesn't tell you about bugs, but it gives some clue what behavior the authors intended. When plugin authors need to do detailed investigation of the implementation, rather than just relying on what was both intended and works today, it makes them more likely to break.